ush/config_defaults.sh

```diff
@@ -505,8 +592,10 @@ SFCOBS_USELIST="/home/amb-verif/ruc_madis_surface/mesonet_uselists"
 #
 EXTRN_MDL_NAME_ICS="FV3GFS"
 EXTRN_MDL_NAME_LBCS="FV3GFS"
+EXTRN_MDL_ICS_OFFSET_HRS="0"
 LBC_SPEC_INTVL_HRS="6"
-EXTRN_MDL_LBCS_OFFSET_HRS=""
+EXTRN_MDL_LBCS_OFFSET_HRS="0"
```

Comment:
This changes the behavior of the logic in `set_extrn_mdl_params.sh`. Please leave blank.

Reply:
This modification changes the default behavior of coldstart, and that is not ideal. All coldstart runs with RAP boundary conditions will be doing something different than previously expected..

Reply:
OK, I set it back to "".